apps/wallet/src/ui/app/components/copy-to-clipboard/index.tsx

```diff
@@ -26,42 +25,17 @@ function CopyToClipboard({
     copyOnlyOnIconClick = false,
     className,
     mode = 'normal',
+    copySuccessMessage,
 }: CopyToClipboardProps) {
-    const [copied, setCopied] = useState(false);
-    const copyToClipboard = useCallback<MouseEventHandler<HTMLElement>>(
-        async (e) => {
-            e.stopPropagation();
-            e.preventDefault();
-            if (!txt) {
-                return;
-            }
-            await navigator.clipboard.writeText(txt);
-            setCopied(true);
-        },
-        [txt]
-    );
-    useEffect(() => {
-        let timeout: number;
-        if (copied) {
-            timeout = window.setTimeout(
-                () => setCopied(false),
-                COPY_CHECKMARK_MILLIS
-            );
-        }
-        return () => {
-            if (timeout) {
-                clearTimeout(timeout);
-            }
-        };
-    }, [copied]);
+    const copyToClipboard = useCopyToClipboard(txt, { copySuccessMessage });
     return (
         <span
             className={cl(st.container, className)}
             onClick={!copyOnlyOnIconClick ? copyToClipboard : undefined}
         >
             {children}
             <Icon
-                className={cl(st.copyIcon, st[mode], { [st.copied]: copied })}
+                className={cl(st.copyIcon, st[mode])}
```

Comment:
can we use Tailwind here

Reply:
We can but this component has other usages of css modules and classes and I think it's better to address them in an another PR that will do only that